app/components/UI/SettingsDrawer/index.js

```diff
@@ -95,7 +76,7 @@ const propTypes = {
   /**
    * Title color
    */
-  titleColor: PropTypes.string,
+  titleColor: TextColor,
```

Comment:
This is not correct, PropTypes only accept PropTypes properties. This is currently causing a warning on main. 
<img width="200" src="https://github.com/MetaMask/metamask-mobile/assets/1024246/cb28ac89-ea77-4f72-a2d8-7e646d7dee98" />

```suggestion
  titleColor: PropTypes.string,
```

Reply:
> This is not correct, PropTypes only accept PropTypes properties. This is currently causing a warning on main. <img alt="" width="200" src="https://private-user-images.githubusercontent.com/1024246/300951763-cb28ac89-ea77-4f72-a2d8-7e646d7dee98.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2NDgzMDIsIm5iZiI6MTcwNjY0ODAwMiwicGF0aCI6Ii8xMDI0MjQ2LzMwMDk1MTc2My1jYjI4YWM4OS1lYTc3LTRmNzItYTJkOC03ZTY0NmQ3ZGVlOTgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMzBUMjA1MzIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDI0OTU5ODdmNzAyYTFkZTUyZTMzMmVkNjQyYWE3OTc4NWJhMDVhODBjZmQ0MzE4NGVmYjc5ODZmMjU5YjYwMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.jEBv7lZW0SXN9vncy72jFjyXjYEvf-0lQq_a73s38CQ">

[PR to fix](https://github.com/MetaMask/metamask-mobile/pull/8476)